index.js

```diff
@@ -38,23 +39,42 @@ module.exports = class SeedBee extends ReadyResource {
 
   async get (key, opts) {
     key = Id.decode(key)
-    const entry = await this.bee.get(key, opts)
+    const entry = await this.bee.get(key, { ...contentEncoding, ...opts })
```

Comment:
`...opts` first so encoding can't be even accidentally messed up by the user: `{ ...opts, ...contentEncoding }`